cpu/esp32/doc.txt

```diff
@@ -166,7 +166,7 @@ either based on
 - Tensilica Xtensa 32-bit LX7 microprocessor (ESP32-S2, ESP32-S3), or
 - 32-bit RISC-V CPU (ESP32-C3, ESP32-H2).
 
-At the moment, ESP32 variant is supported by RIOT-OS.
+At the moment, ESP32 and ESP32-C3 variants (families) are supported by RIOT-OS.
```

Comment:
Now this is a lie until #18345 is merged :wink:

Reply:
Yes, but I think this is acceptable in a development branch for a short time. Initially I thought we could merge PR #18345 first and then this PR until I noticed that a reference to the documentation is used in the header files of PR #18345 :worried: